### PR TITLE
Issue #3056095: Update JS libraries

### PR DESCRIPTION
**Pull request by chrfritsch**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-3.x/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -160,9 +160,7 @@
         "npm-asset/jqueryui": "*",
         "npm-asset/backbone": "*",
         "npm-asset/underscore": "*",
-        "npm-asset/fortawesome--fontawesome-free-webfonts": "*",
-        "npm-asset/jquery-mousewheel": "*",
```
Comment (chrfritsch) [inline composer.json:164]:
```
        "npm-asset/almond": "*",
        "npm-asset/jquery-mousewheel": "*",
```
These are now dev-dependencies of select2

File: composer.json [hunk 2]
```diff
@@ -160,9 +160,7 @@
...
-        "npm-asset/almond": "*"
+        "npm-asset/fortawesome--fontawesome-free": "*"
```
Comment (chrfritsch) [inline composer.json:163]: Shariff changed it's dependency from "npm-asset/fortawesome--fontawesome-free-webfonts" to "npm-asset/fortawesome--fontawesome-free"

File: drupal-org.make [hunk 3]
```diff
@@ -148,4 +148,4 @@ libraries[slick_lib][download][url] = https://github.com/kenwheeler/slick/archiv
 libraries[pusher][type] = library
 libraries[pusher][directory_name] = pusher
 libraries[pusher][download][type] = get
-libraries[pusher][download][url] = https://github.com/pusher/pusher-http-php/archive/v3.4.0.zip
+libraries[pusher][download][url] = https://github.com/pusher/pusher-http-php/archive/v3.4.1.zip
```
Comment (chrfritsch) [inline drupal-org.make:151]: Adds PHP 7.3 Support

**Review by dbosen — APPROVED** [review]
fine

**Pull request merged**